### PR TITLE
Cloudfront redirect to https

### LINKED ISSUE #146: V10 - CloudFront should upgrade to https
Currently the policy is set to `https_only`

https://github.com/cds-snc/covid-shield-server/blob/master/config/terraform/aws/cloudfront.tf#L37

Connections on port 80 are returning a 403 error. For consistencies sake we should probably close port 80 by upgrading the connection using `redirect-to-https`

### PR DESCRIPTION
**Pull request by whytoe**
Cloudfront to redirect to HTTPS closes #146

### CODE REVIEW

File: config/terraform/aws/cloudfront.tf [hunk 1]
```diff
@@ -10,7 +10,7 @@ resource "aws_cloudfront_distribution" "key_retrieval_distribution" {
     custom_origin_config {
       http_port              = 80
       https_port             = 443
-      origin_protocol_policy = "https-only"
+      origin_protocol_policy = "http-only"
```
Comment (stonith) [inline config/terraform/aws/cloudfront.tf:13]: Missed this earlier, this should not be changed to http

Comment (whytoe) [reply config/terraform/aws/cloudfront.tf]: thanks, I talked to @maxneuvians I thought origin was client side, not destination, Ive since fixed it

**Comment by maxneuvians** [comment]
@obrien-j This is partially to satisfy:

`6.1.1 is configured for HTTPS (and redirected from HTTP)`

from https://www.canada.ca/en/government/system/digital-government/modern-emerging-technologies/policy-implementation-notices/implementing-https-secure-web-connections-itpin.html#toc6

I agree with @stonith that it does not increase security, but need a call from you in regards to #146 and if the policy in fact demands this.

**Comment by obrien-j** [comment]
The policy doesn’t apply to api’s in our example so we are all good just
turning off port 80.

**Pull request closed without merging**